package-configuration/linux-cli/get-cloudify.py

```diff
@@ -425,9 +438,12 @@ def main():
     installer = CloudifyInstaller(args)
     installer.execute()
     if args.virtualenv:
-        prn('You can now run: "source {0}" to activate the Virtualenv.'.format(
-            os.path.join(args.virtualenv, 'bin/activate')))
+        lgr.info('You can now run: "source {0}" to activate '
+                 'the Virtualenv.'.format(
+                     os.path.join(args.virtualenv, 'bin/activate')))
+
 
+lgr = init_logger('get-cloudify')
```

Comment:
Also, I didn't use __name__ because I don't want the logger to be named __main__.